src/index.js

```diff
@@ -3,13 +3,20 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import request from './api-requests.js';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+let movies = [];
+
+request.getMovieData()
```

Comment:
This strikes me as a good place for the request to live! I can check with my mentor in our meeting tomorrow, just to make sure we've got good practices and all that, but having it being in `index.js` seems like a swell idea to me.